src/Serilog.Sinks.Email/LoggerConfigurationEmailExtensions.cs

```diff
@@ -215,15 +221,18 @@ public static class LoggerConfigurationEmailExtensions
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
             int batchPostingLimit = EmailSink.DefaultBatchPostingLimit,
             TimeSpan? period = null,
-            string mailSubject = EmailConnectionInfo.DefaultSubject)
+            ITextFormatter mailSubjectFormatter = null)
```

Comment:
The body `textFormatter` is there so that the body can be text, JSON, whatever - I don't think it's as useful for subject. Could we avoid a breaking change by switching back to string here?